docs/releasing.md

```diff
@@ -81,6 +78,10 @@ You don't have to do anything special to release the documentation. GitHub pages
 * Title the release `x.y.z`, optionally `x.y.z - pithy summary`
 * Add release notes, linking pull requests included in this release, linking the newly released documentation, and especially articulating anything interesting about the upgrade path to this release for frame-based app developers.
 
+### Update My App Seed
+
+[My App Seed][my-app-seed] is a template for developers to use when creating new frame apps. When cutting a release, be sure to update the [pom.xml][my-app-seed pom] so future adopters can start developing using the latest code.
```

Comment:
:speaking_head: plain `pom.xml` looks like an internal link. For better link accessibility

``` markdown
[my-app-seed pom][]
```